src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -201,9 +201,11 @@ def create_connectedk8s(cmd, client, resource_group_name, cluster_name, https_pr
 
     # Resource group Creation
     if resource_group_exists(cmd.cli_ctx, resource_group_name, subscription_id) is False:
-        resource_group_params = {'location': location}
+        from azure.cli.core.profiles import ResourceType
+        ResourceGroup = cmd.get_models('ResourceGroup', resource_type=ResourceType.MGMT_RESOURCE_RESOURCES)
```

Comment:
There is another place where we use ResourceType.MGMT_RESOURCE_RESOURCES.
https://github.com/Azure/azure-cli-extensions/blob/1010d2af2fe867a9b361d30db8f3f44e4868816a/src/connectedk8s/azext_connectedk8s/custom.py#L382
imported from: https://github.com/Azure/azure-cli-extensions/blob/1010d2af2fe867a9b361d30db8f3f44e4868816a/src/connectedk8s/azext_connectedk8s/_client_factory.py#L32

Reply:
Yes, I know it. However, there is no breaking change in this logic, so no additional compatibility logic is required.

Reply:
For the breaking changes in the new `azure-mgmt-resource` SDK, please refer to the description of PR https://github.com/Azure/azure-cli/pull/17783